### PR TITLE
DCOS-50468: Nodes Table Loader does not behave as expected

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Change the initial value for nodes to null
to show loader while data is loading.

Closes https://jira.mesosphere.com/browse/DCOS-50468

## Testing
1. Go to Nodes tab
2. Refresh the page
3. Verify that an empty table is not visible as data is loading

## Trade-offs
None.

## Dependencies
None.

### Before
Peek 2019-03-20 11-42

### After
Peek 2019-03-20 15-41

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
@natmegs , good catch. I fixed it.
The failing integration test is flaky.

File: plugins/nodes/src/js/components/NodesTable.tsx [hunk 1]
```diff
@@ -155,7 +163,7 @@ export default class NodesTable extends React.Component<
   componentWillReceiveProps(nextProps: NodesTableProps): void {
     this.setState(
       this.updateData(
-        nextProps.hosts.getItems(),
+        nextProps.hosts ? nextProps.hosts.getItems() : null,
```
Comment (nLight) [inline plugins/nodes/src/js/components/NodesTable.tsx]: do we even need updating data when we don't get `nextProps.hosts`?

Comment (nLight) [reply plugins/nodes/src/js/components/NodesTable.tsx]: sorting comes from state 🤔

Comment (GeorgiSTodorov) [reply plugins/nodes/src/js/components/NodesTable.tsx]: I can imagine some situations where we do want to update even when `nextProps.hosts` is null or undefined or empty.

Comment (nLight) [reply plugins/nodes/src/js/components/NodesTable.tsx]: Ok, I see 1 case at least. When for some reason `props.hosts` turn to undefined we want to `nullify` the `state.data`  thus showing loader.

**Pull request merged**